### ISSUE TITLE
Suggestion: Simple example command in instructions.

### ISSUE THREAD

**Issue by patrickplaggenborg** [vzhd1701/enex2notion#4]
I am somewhat comfortable using terminal commands, but creating the right command from the full specification requires some extra thought and time.

I love how you provided a simple example in the Reddit post about `enex2notion`:
https://www.reddit.com/r/Notion/comments/r5nnry/alternative_way_of_importing_notebooks_from/

Your example command:
```python
$ enex2notion --token <YOUR_TOKEN_HERE> "my_notebooks/Test Notebook.enex"
```

**Suggestion:**
It would be super helpful to have this example command in the instructions here on GitHub as well.

**Comment by vzhd1701** [comment]
I have a couple of examples [below the usage section](https://github.com/vzhd1701/enex2notion#examples). Do you mean I should replace `--token "30d0...9c12"` in README example with `--token <YOUR_TOKEN_HERE>` to make it more readable?

**Comment by patrickplaggenborg** [comment]
I think YOUR_TOKEN_HERE was indeed easier for me to read because the ... was confusing me a bit.

I indeed just saw the examples and wondered why I didn't see it before. I did not immediately notice the command example in the code block because it contained both the command and the feedback after.

My suggestion was mostly about displaying the example command as a single line code block, similar to how you do it in the Installation section. That section is really clear and succinct because it visually clearly separates:
- Title
- Explanation
- Command

The full log as an example of usage is still helpful. I would suggest displaying it as a separate code block under the single command.
